assistant/cyton/cyton_controller.py

```diff
@@ -65,15 +68,71 @@ def set_pose(self, q: SE3):
                 self.sock.send(q)
         # TODO: fake printouts
 
-    def go_home(self):
-        self.set_pose(self.robot.qz)
 
-    def grab_object(self, item: BaseItem):
-        robot_object_pose = self.robot.ikine_LM(item.pose, q0=self.pose)
+    def open_gripper(self):
+        """
+        Opens the gripper
+        :return: None
+        """
+        self.robot.open_gripper()
+
+
+    def close_gripper(self):
+        """
+        Closes the gripper
+        :return: None
+        """
+        self.robot.close_gripper()
+
+
+    def goto_pickup(self, item: BaseItem):
+        """
+        Goes to the pickup location
+        :return: None
+        """
+        self.open_gripper()     # open gripper to pick up object (or keep gripper open)
 
-        traj = jtraj(self.pose, robot_object_pose, 100)
+        robot_pickup_pose = self.robot.ikine_LM(item.pose, q0=self.pose)
+        traj = jtraj(self.pose, robot_pickup_pose, 100)
 
+        self.set_pose(traj)
```

Comment:
I've been thinking about this a bit, let's make a function in the controller that runs trajectories. Something like

```python
def run_trajectory(self, qs: List[List]]):
    for q in qs:
         self.set_pose(q)
```